homeassistant/components/ohme/coordinator.py

```diff
@@ -0,0 +1,65 @@
+"""Ohme coordinators."""
+
+from datetime import timedelta
+import logging
+
+from ohme import ApiException, AuthException, OhmeApiClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+_LOGGER = logging.getLogger(__name__)
+
+type OhmeConfigEntry = ConfigEntry[OhmeCoordinator]
+
+
+class OhmeCoordinator(DataUpdateCoordinator[OhmeApiClient]):
```

Comment:
```suggestion
class OhmeCoordinator(DataUpdateCoordinator[None]):
```

only noticed that now, but since you're storing the client in the coordinator anyway and that is storing the data, you do not need to return anything during the update